src/Microsoft.Tye.Hosting/TyeHost.cs

```diff
@@ -112,8 +111,16 @@ private static WebApplication BuildWebApplication(Application application, HostO
             // Logging for this application
             builder.Host.UseSerilog((context, configuration) =>
             {
+                var logLevel = options.LogVerbosity switch
+                {
+                    Verbosity.Quiet => LogEventLevel.Warning,
+                    Verbosity.Info => LogEventLevel.Information,
+                    Verbosity.Debug => LogEventLevel.Verbose,
+                    _ => default
```

Comment:
What's the default?

Reply:
You mean `_ => default`?
This is to appease the C# compiler :)
It throws a warning otherwise

Reply:
I’m asking why use default and not an end value like Verbose or Info for the default case

Reply:
That `_ => default` should never happen. It's just an artifact of the compiler treating the enum as an integer. There is already a place where we set the default value for the verbosity, so I didn't want to repeat that in multiple places.

Reply:
Ah, I got it.